autoscale-api/src/main/java/com/sequenceiq/periscope/api/model/TriggerStatus.java

```diff
@@ -0,0 +1,19 @@
+package com.sequenceiq.periscope.api.model;
+
+public enum TriggerStatus {
+    METRICS_COLLECTION_SUCCESS,
+    METRICS_COLLECTION_FAILED,
+
+    MANDATORY_UPSCALE,
```

Comment:
Do we need MANDATORY_UPSCALE/MANDATORY_DOWNSCALE as Status fields here? as against something like SCALING_STARTED (or UPSCALE_STARTED, DOWNSCALE_STARTED)?

Reply:
Removed these fields, for now, will add more fields based on how new scalingActivity records are to be populated